test/showcase/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+load(
+    "//rules_java_gapic:java_gapic.bzl",
+    "java_gapic_library",
+    "java_gapic_test",
+)
+load("@io_grpc_grpc_java//:java_grpc_library.bzl", "java_grpc_library")
+load("@rules_gapic//:gapic.bzl", "proto_library_with_info")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//rules_java_gapic:java_gapic_pkg.bzl", "java_gapic_assembly_gradle_pkg")
+
+proto_library_with_info(
+    name = "showcase_proto_with_info",
+    deps = [
+        "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto",
+        "@com_google_googleapis//google/cloud:common_resources_proto",
+    ],
+)
+
+java_proto_library(
+    name = "showcase_java_proto",
+    deps = ["@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto"],
+)
+
+java_grpc_library(
+    name = "showcase_java_grpc",
+    srcs = ["@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto"],
+    deps = [":showcase_java_proto"],
+)
+
+java_gapic_library(
+    name = "showcase_java_gapic",
+    srcs = [":showcase_proto_with_info"],
+    gapic_yaml = None,
+    grpc_service_config = "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_grpc_service_config.json",
+    service_yaml = "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_v1beta1.yaml",
+    test_deps = [
+        ":showcase_java_grpc",
+    ],
+    transport = "grpc+rest",
+    deps = [
+        ":showcase_java_proto",
+        "@com_google_googleapis//google/api:api_java_proto",
+    ],
+)
+
+# Open Source Packages
+java_gapic_assembly_gradle_pkg(
+    name = "google-cloud-showcase-v1beta1-java",
+    include_samples = True,
```

Comment:
We don't need to generate samples for showcase.

Reply:
Done